src/coreclr/jit/gentree.h

```diff
@@ -4037,6 +4048,16 @@ struct GenTreeField : public GenTreeUnOp
         return (gtFlags & GTF_FLD_VOLATILE) != 0;
     }
 
+    bool IsNeverNegative() const
+    {
+        return gtFldIsNeverNegative;
+    }
+
+    void SetIsNeverNegative(bool value)
```

Comment:
The boolean parameter is always passed the constant `true`. Delete?

It would also be more consistent with the prevailing convention for such setters (I realize not all of them follow it, just the majority, see e.g. `GenTreeCall`).

Same applies for the `GenTreeField` version.

Reply:
It is always passed as true today, that may not always be the case in the future.

This pattern matches how a typical `get/set` property would function and ensures we don't need multiple different code paths to do the same simple assignment.